frontend/src/features/landing_page/components/Team.tsx

```diff
@@ -72,6 +72,11 @@ const Team = () => {
             role="Software Engineer"
             path="/Avatars/Paul_Buschmann.jpg"
           />
+          <Teammember
+            name="Christoph Kraus"
+            role="Software Engineer"
```

Comment:
Maybe something related to Database or Security that we don't have the same everywhere?